Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -853,9 +853,20 @@ void msblenContext::doExtractNonEditMeshData(msblenContextState& state, BlenderS
     const size_t num_polygons = polygons.size();
     size_t num_vertices = vertices.size();
 
-    std::vector<int> mid_table(mesh.totcol);
-    for (int mi = 0; mi < mesh.totcol; ++mi)
-        mid_table[mi] = getMaterialID(mesh.mat[mi]);
+    std::vector<int> mid_table(bobj.m_ptr->totcol);
```

Comment:
is bobj.m_ptr-totcol guaranteed to be the same as mesh.totcol?

Reply:
Yeah, this is what the blender code uses.